ruby/rails6-mongo/app/Gemfile

```
@@ -0,0 +1,52 @@
+source 'https://rubygems.org'
+git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+ruby '3.0.3'
+
+gem 'mongoid', '~> 7.0.5'
```

Can we use the latest mongoid version?

```suggestion
gem 'mongoid', '~> 7.3.3'
```

---

Sure, this one is from their docs